sample/mpp-library/src/commonMain/kotlin/com/icerockdev/library/SampleViewModel.kt

```diff
@@ -0,0 +1,44 @@
+package com.icerockdev.library
+
+import dev.icerock.moko.mvvm.dispatcher.EventsDispatcher
+import dev.icerock.moko.mvvm.dispatcher.EventsDispatcherOwner
+import dev.icerock.moko.mvvm.viewmodel.ViewModel
+import dev.icerock.moko.permissions.Permission
+import dev.icerock.moko.permissions.PermissionsController
+import dev.icerock.moko.permissions.DeniedException
+import dev.icerock.moko.permissions.DeniedAlwaysException
+import kotlinx.coroutines.launch
+
+class SampleViewModel(
+    override val eventsDispatcher: EventsDispatcher<EventListener>,
+    private val permissionsController: PermissionsController
+) : ViewModel(), EventsDispatcherOwner<SampleViewModel.EventListener> {
+
+    /**
+     * An example of using [PermissionsController] in common code.
+     */
+    fun onRequestPermissionButtonPressed(permission: Permission) {
```

Comment:
which permission should be requested should define viewmodel, not view (for clear sample in mvvm)